test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -237,6 +237,18 @@ func testSecondaryNetwork(t *testing.T, networkType string, pods []*testPodInfo)
 			t.Fatalf("Error when pinging between interfaces: %v", err)
 		}
 	})
+	t.Run("testRetstartAntreaAgentPods", func(t *testing.T) {
+		if err := e2eTestData.RestartAntreaAgentPods(defaultTimeout); err != nil {
+			t.Fatalf("Failed to restart Antrea agent pods within timeout %v: %v", defaultTimeout, err)
+		}
+	})
+	// Again verify interfaces restoration and connnectivity
+	t.Run("testpingBetweenInterfaces", func(t *testing.T) {
+		err := testData.pingBetweenInterfaces(t)
```

Comment:
Could we also check secondary OVS ports can be removed after Pods are removed? Can be better if we can check IPs can be released too.